modin/pandas/groupby.py

```diff
@@ -1196,19 +1245,28 @@ def _default_to_pandas(self, f, *args, **kwargs):
         by = GroupBy.validate_by(by)
 
         def groupby_on_multiple_columns(df, *args, **kwargs):
-            return f(
-                df.groupby(
-                    by=by, axis=self._axis, squeeze=self._squeeze, **self._kwargs
-                ),
-                *args,
-                **kwargs,
+            groupby_obj = df.groupby(
+                by=by, axis=self._axis, squeeze=self._squeeze, **self._kwargs
             )
 
+            if callable(f):
+                return f(groupby_obj, *args, **kwargs)
+            else:
+                ErrorMessage.catch_bugs_and_request_email(
+                    failure_condition=not isinstance(f, str)
+                )
+                attribute = getattr(groupby_obj, f)
+                if callable(attribute):
+                    return attribute(*args, **kwargs)
+                return attribute
```

Comment:
Isn't this covered by a test?

Reply:
We only default-to-pandas [on callable attributes](https://github.com/dchigarev/modin/blob/b5962d0be71bbfef3aafa187aa3821a8b4fa7963/modin/pandas/groupby.py#L159), so yeah, this line is untested as we can't end-up here with an `f` defining a property for now.